src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -508,7 +508,7 @@ export class SVGViewer extends React.Component<SvgViewerProps, SvgViewerState> {
 
   zoomOnCurrentAsset = (currentAsset: Element | null) => {
     const isDesktop = this.state.width > minDesktopWidth;
-    if (!currentAsset || !this.pinchZoomInstance.container) {
+    if (!currentAsset || !this.pinchZoomInstance) {
```

Comment:
Won't line 524 `this.pinchZoomInstance.container.clientWidth` fail with type error if you are not checking `this.pinchZoomInstance.container` against `undefined`?

Reply:
no, I guess, cause `PinchZoomInterface` interface describes `container` as non-optional property with `HTMLElement` type.

Reply:
Good point as long as the type is correct